privacy.html

```diff
@@ -117,15 +117,6 @@
 
 <style>
 	#privacy_optin, #privacy_optout {
-		opacity: 1;
 		-webkit-appearance: auto;
```

Comment:
Unsure of browser support on prefixes but it looks like maybe should use either `appearance` or use all three vendor prefixes to be safe: https://css-tricks.com/almanac/properties/a/appearance/